lib/shell/shellscript.js

```diff
@@ -50,8 +50,11 @@ class ShellScriptWorker {
 
         const preparedFiles = await prepareMetadata(rendition.directory);
 
+        // rendition options file for optiosn like postProcess
+        const optionsFile = path.resolve(rendition.directory, 'options.json');
+
         // inherit environment variables
-        const env = setVariables(rendition, source, preparedFiles.errorFile, preparedFiles.typeFile);
+        const env = setVariables(rendition, source, preparedFiles.errorFile, preparedFiles.typeFile, optionsFile);
```

Comment:
At that point, I would consider putting the various files to create in an object, to avoid keeping changing the `setVariables` signature.

Reply:
yeah, well hopefully this `optionsFile` will cover any remaining things we would write to files...so no more variables